### PR TITLE
Windows support

### PR DESCRIPTION
**Pull request by generall**
Fixes for windows:
- disable pprof
- minor fixes in test
- update CI
- use compatible WAL

### CODE REVIEW

**Comment by generall** [comment]
Tested with `bfb` as well

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -22,8 +25,8 @@ jobs:
         toolchain: stable
     - uses: actions/checkout@v3
     - uses: Swatinem/rust-cache@v2
-    - name: Install dependencies
-      run: sudo apt-get install clang protobuf-compiler
+    - name: Install Protoc
+      uses: arduino/setup-protoc@v1
```
Comment (IvanPleshkov) [inline .github/workflows/rust.yml:29]: There are explicit dependencies. `Clang` isn't mentioned but it's required for C++ dependencies like rocksdb

**Comment by agourlay** [comment]
WAL PR merged :heavy_check_mark:

File: .github/workflows/rust.yml (hunk 1 above)
Comment (generall) [reply .github/workflows/rust.yml:29]: Somehow it works :confused: I think clang is installed somewhere in rust installation step

Comment (IvanPleshkov) [reply .github/workflows/rust.yml:29]: Let's keep it as is and fix it when something will go wrong

**Pull request merged**